README.md

```diff
@@ -122,3 +122,26 @@ and then run `cargo build` (this uses the `dev` [profile](https://doc.rust-lang.
 ## Static linking on Linux
 
 We provide a convenient Dockerfile to build a static binary. You can just run `make linux-static` and it will build a Docker image named `asfd-build` based on the official Rust Alpine container image. It will then use that image to build a static binary. You can choose the build [profile](https://doc.rust-lang.org/cargo/reference/profiles.html) with the `PROFILE` variable, eg `PROFILE=release make linux-static` and the binary will then be found at `target/debug/asfd`.
+
+# Contributing
+
+We welcome contributions of all kinds! To ensure code quality and consistency, this project uses [pre-commit](https://pre-commit.com/) hooks to automatically check and format code before it's committed.
```

Comment:
Can you link to the blog post about contributions? Eg
`[contributions](https://www.asfaload.com/blog/handling-outside-contributions/)`